README.md

```diff
@@ -425,6 +425,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Age of Empires II](https://age-of-empires-2-api.herokuapp.com) | Get information about Age of Empires II resources | No | Yes | No |
 | [AmiiboAPI](https://amiiboapi.com/) | Nintendo Amiibo Information | No | Yes | Yes |
+| [Autochess VNG API](https://github.com/didadadida93/autochess-vng-api) | Rest Api for Autochess VNG | No | Yes | Yes |
```

Comment:
The validation format check felt here: 
<img width="1399" alt="Screenshot 2021-04-29 at 15 30 29" src="https://user-images.githubusercontent.com/1914165/116568100-1dccfe80-a900-11eb-90a9-fbad885034de.png">

Can you please remove `API` from the title?